services/credential-server/src/utils/lmdb.ts

```diff
@@ -0,0 +1,39 @@
+import { open, RootDatabase } from "lmdb";
+
+class Lmdb {
+  private static instance: Lmdb;
+  private db: RootDatabase;
+
+  private constructor() {
+    this.db = open({
+      path: "./data/lmdb",
+    });
+  }
+
+  public static getInstance(): Lmdb {
```

Comment:
Not too important but no need to put `public` as its default. You can also use `static get instance` so it's more like `db.instance` instead of `db.getInstance()`